### ISSUE TITLE
Sidebar a11y updates: <aside>, aria-label

### ISSUE THREAD

**Issue by dbranchini** [sul-dlss/earthworks#1437]
Because we have a sidebar on the left, the H1 is not seen/read first. We discussed this with SODA and they agreed that a left sidebar is a very common pattern (and in fact used on the WCAG site). Also, it appears under the category of "best practice" in SiteImprove, so it's not as important as an A or AA error. It also doesn't show up in Axe.

All that said, SODA suggested it would be improved by making the entire sidebar area an `<aside>` and giving it an aria-label that describes it, such as "Download, export and other options."

<img width="271" alt="Screenshot 2024-10-16 at 5 06 42 PM" src="https://github.com/user-attachments/assets/84b2d269-2d02-41e7-84c2-23b7d98457ff">

**Comment by dbranchini** [comment]
Also, if it's easy enough, "Also available at" and "Item belong to a collection" should be in the same gray box. See this Figma design - https://www.figma.com/design/nkgg905s5FyXoyPZjVMco0/EarthWorks?node-id=517-18605&t=N5nCgaAU3Yv8eOEv-4.

**Comment by thatbudakguy** [comment]
Opened https://github.com/projectblacklight/blacklight/pull/3423 in blacklight to partially address this.

**Comment by thatbudakguy** [comment]
Spacing is now fixed; waiting on Blacklight release to make it a labeled `<aside>`.

![Screenshot 2024-11-04 at 08 58 03](https://github.com/user-attachments/assets/e239036e-7783-4d75-bf34-801e2cf2f870)

**Comment by thatbudakguy** [comment]
I forgot to backport this 🤦🏻 opened https://github.com/projectblacklight/blacklight/pull/3458 to get it into blacklight 8.x.
